.github/workflows/lint.yml

```diff
@@ -1,5 +1,21 @@
 name: Lint
 
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  id-token: none
+  issues: none
+  discussions: none
+  packages: none
+  pages: none
+  pull-requests: none
```

Comment:
Do we not want this enabled somehow so that this workflow can work on pull requests?

Reply:
The workflow will still be triggered on pull requests, just the job itself won't be able to mutate the pull request - which is intended functionality.